### PR TITLE
Skip test_replace_fec for known issue

### PR DESCRIPTION
**Pull request by isabelmsft**
### Description of PR

Summary:
Fixes # (issue)

### Type of change

- [ ] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [ ] Test case(new/improvement)

### Back port request
- [ ] 201911
- [ ] 202012
- [ ] 202205
- [x] 202305
- [x] 202311

#### What is the motivation for this PR?
Skip test_replace_fec because it will cause testbed unhealthy on 3800 and 7260 devices (issue tracked in https://github.com/sonic-net/sonic-mgmt/issues/11237)

#### How did you do it?
Skip in conditional mark file

### CODE REVIEW

File: tests/common/plugins/conditional_mark/tests_mark_conditions.yaml [hunk 1]
```diff
@@ -611,7 +611,7 @@ generic_config_updater/test_ecn_config_update.py::test_ecn_config_updates:
       - "topo_type in ['m0', 'mx']"
       - "release in ['202211']"
 
-generic_config_updater/test_eth_interface.py::test_eth_interface_admin_change:
```
Comment (qiluo-msft) [inline tests/common/plugins/conditional_mark/tests_mark_conditions.yaml:614]: Do you still need to skip old testcase based on old issue?

Comment (ZhaohuiS) [reply tests/common/plugins/conditional_mark/tests_mark_conditions.yaml:614]: @qiluo-msft yes, the issue is same, actually, it's caused by test_replace_fec not test_eth_interface_admin_change. I verified on 7260 t0 testbed.

**Review by ZhaohuiS — APPROVED** [review]
LGTM

**Review by StormLiangMS — APPROVED** [review]
LGTM

**Pull request merged**